metaci/plan/forms.py

```diff
@@ -16,37 +16,52 @@ class RunPlanForm(forms.Form):
     branch = forms.ChoiceField(choices=(), label='Branch')
     commit = forms.CharField(required=False)
     keep_org = forms.BooleanField(required=False)
+    release = forms.ModelChoiceField(None, required=False)
 
     def __init__(self, plan, repo, user, *args, **kwargs):
         self.plan = plan
         self.repo = repo
         self.user = user
         super(RunPlanForm, self).__init__(*args, **kwargs)
         self.fields['branch'].choices = self._get_branch_choices()
+        self.fields['release'].queryset = self.repo.releases
         self.helper = FormHelper()
         self.helper.form_class = 'form-vertical'
         self.helper.form_id = 'run-build-form'
         self.helper.form_method = 'post'
+        self.advanced_mode = False
+        if 'advanced_mode' in args[0]:
+            self.advanced_mode = (args[0]['advanced_mode'] == "1")
```

Comment:
Can we include a link to switch to advanced mode somewhere on the page? I think it's fine to have an advanced mode triggered by a URL param, but I'd rather it not be hidden from anyone who wasn't told or has forgotten it exists.